sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOTest.java

```diff
@@ -126,9 +126,15 @@ public static void startCassandra() throws Exception {
 
   @AfterClass
   public static void stopCassandra() {
-    EmbeddedCassandraServerHelper.cleanEmbeddedCassandra();
-    session.close();
-    cluster.close();
+    if (cluster != null && session != null) {
+      EmbeddedCassandraServerHelper.cleanEmbeddedCassandra();
+      session.close();
```

Comment:
shouldn't this be done before cleaning the embedded server?

Reply:
no because cleanEmbeddedCassandra require both the cluster and the session to drop the keyspace